.vscode/settings.json

```diff
@@ -1,8 +1,18 @@
 {
     "java.configuration.updateBuildConfiguration": "automatic",
+    "eslint.workingDirectories": [ "./client" ],
+    "eslint.options": {
+      "extensions": [".ts", ".html"]
```

Comment:
I assume `".js"` is implied and doesn't need to be explicitly included?

Reply:
Actually we probably don't need this at all.